root/roofitstats/vectorisedPDFs/CMakeLists.txt

```diff
@@ -6,27 +6,46 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL Intel AND DEFINED ENV{VTUNE_AMPLIFIER_2019_DIR
 endif()
 
 add_library(VectorisedPDFTests STATIC VectorisedPDFTests.cxx)
-link_libraries(VectorisedPDFTests Gpad RooFitCore RooFit)
-add_dependencies(VectorisedPDFTests RooFitCore)
+target_link_libraries(VectorisedPDFTests Gpad RooFitCore RooFit)
```

Comment:
This is better, but will only work if you use `ROOT::...` for the library names.

Reply:
Is listing the libraries preferred?

Reply:
Either is fine, using the targets is probably easier.